app/controllers/workflow/pages_controller.rb

```diff
@@ -6,186 +6,186 @@ class Workflow::PagesController < ApplicationController
-          if target_user
-            email_blank_users.push(target_user.name) if target_user.email.blank?
-          end
-        end
-        target_user = nil
-        target_user_id = @item.workflow_user_id || @cur_user._id
-        target_user = SS::User.find(target_user_id) rescue nil
+        target_user = SS::User.find(workflow_approver[:user_id]) rescue nil
         if target_user
           email_blank_users.push(target_user.name) if target_user.email.blank?
         end
       end
-      email_blank_users.uniq!
-      email_blank_users.sort!
-      return nil if email_blank_users.blank?
-      message = t("errors.messages.user_email_blank")
-      email_blank_users.each do |email_blank_user|
-        message += "\n#{email_blank_user}"
+      target_user = nil
+      target_user_id = @item.workflow_user_id || @cur_user._id
+      target_user = SS::User.find(target_user_id) rescue nil
+      if target_user
+        email_blank_users.push(target_user.name) if target_user.email.blank?
       end
-      message
     end
+    email_blank_users.uniq!
+    email_blank_users.sort!
+    return nil if email_blank_users.blank?
+    message = t("errors.messages.user_email_blank")
+    email_blank_users.each do |email_blank_user|
+      message += "\n#{email_blank_user}"
+    end
+    message
+  end
 
   public
 
-    def request_update
-      raise "403" unless @item.allowed?(:edit, @cur_user)
+  def request_update
+    raise "403" unless @item.allowed?(:edit, @cur_user)
 
-      if params[:forced_update_option] == "false"
-        if message = workflow_alert
-          render json: { workflow_alert: message }
-          return
-        end
+    if params[:forced_update_option] == "false"
+      if message = workflow_alert
+        render json: { workflow_alert: message }
+        return
       end
+    end
 
-      @item.workflow_user_id = @cur_user._id
-      @item.workflow_state   = "request"
-      @item.workflow_comment = params[:workflow_comment]
-      @item.workflow_approvers = params[:workflow_approvers]
-      @item.workflow_required_counts = params[:workflow_required_counts]
-
-      if @item.update
-        request_approval
-        render json: { workflow_state: @item.workflow_state }
-      else
-        render json: @item.errors.full_messages, status: :unprocessable_entity
-      end
+    @item.workflow_user_id = @cur_user._id
+    @item.workflow_state   = "request"
+    @item.workflow_comment = params[:workflow_comment]
+    @item.workflow_approvers = params[:workflow_approvers]
+    @item.workflow_required_counts = params[:workflow_required_counts]
+
+    if @item.update
+      request_approval
+      render json: { workflow_state: @item.workflow_state }
+    else
+      render json: @item.errors.full_messages, status: :unprocessable_entity
     end
+  end
 
-    def approve_update
-      raise "403" unless @item.allowed?(:approve, @cur_user)
+  def approve_update
```

Comment:
__[rubocop]__
* Perceived complexity for approve_update is too high. [16/14]

<!-- Comment by [SideCI](https://sideci.com) -->